python/ray/serve/_private/benchmarks/common.py

```diff
@@ -43,12 +62,12 @@ async def run_throughput_benchmark(
     # real run
     stats = []
     for _ in range(num_trials):
-        start = time.time()
+        start = time.monotonic()
```

Comment:
we should standardize on `time.perf_counter()` everywhere (it's also monotonic but uses highest available resolution)